spec/factories.rb

```diff
@@ -1,14 +1,19 @@
 FactoryBot.define do
   factory :family do
-    last_name "Liso"
-    capability 3
+    last_name { FFaker::NameIT.last_name }
```

Comment:
I am not a fan of using another Gem just to give it random data.
The specs will run faster without it.

Reply:
Ok, I will use self made random data without FFaker in the rake task! :)

Reply:
We can work on this tomorrow